spec/rentals_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+require './book'
+require './rentals'
+
+describe 'Test rental class implementation' do
+  before(:each) do
+    @book = Book.new('Gamers', 'Addis')
+    @person = Person.new('Promise', 30)
+    @rentals = Rental.new('22/01/2020', @person, @book)
+  end
+
+  it 'date of rental implementation' do
+    expect(@rentals.date).to eql '22/01/2020'
+  end
```

Comment:
- [OPTIONAL] 🤔 What if you add more tests here since we have the date, book, and person instance variables 🙃 

```ruby
  def initialize(date, person, book)
    @date = date
    @person = person
    person.rentals << self
    @book = book
    book.rentals << self
  end
end
```

Reply:
Thank you for the feedback. All has ben implemented.